armi/nuclearDataIO/cccc/isotxs.py

```diff
@@ -268,6 +268,47 @@ def readWrite(self):
         .. impl:: Tool to read and write ISOTXS files.
             :id: I_ARMI_NUCDATA_ISOTXS
             :implements: R_ARMI_NUCDATA_ISOTXS
+
+            Reading and writing ISOTXS files is performed using the general
+            nuclear data I/O functionalities described in
+            :need:`I_ARMI_NUCDATA`. A ISOTXS file specifically follows the
+            following steps:
+
+            #. Read/write file ID record
+            #. Read/write file 1D record, which includes:
+
+                * Number of energy groups (``NGROUP``)
+                * Maximum number of up-scatter groups (``MAXUP``)
+                * Maximum number of down-scatter groups (``MAXDN``)
+                * Maximum scattering order (``MAXORD``)
+                * File-wide specification on fission spectrum type, i.e. vector
+                  or matrix (``ICHIST``)
+                * Maximum number of blocks of scattering data (``MSCMAX``)
+                * Subblocking control for scatter matrices (``NSBLOK``)
+
+            #. Read/write file 2D record, which includes:
+
+                * Library IDs for each isotope (``HSETID(I)``)
+                * Isotope names (``HISONM(I)``)
+                * Global fission spectrum (``CHI(J)``) if file-wide spectrum is
+                  specified (``ICHIST`` = 1)
+                * Energy group structure (``EMAX(J)`` and ``EMIN``)
+                * Locations of each nuclide record in the file (``LOCA(I)``)
+
+                    .. note::
+
+                        This is not used within ARMI, because it can compute it
```

Comment:
I would say "The offset data is not read from the binary file because the ISOTXS reader can dynamically calculate the offset itself. Therefore, during a read operation, this data is ignored."